_includes/comments.html

```diff
@@ -166,6 +166,7 @@ <h4 class="page__comments-title">{{ site.data.ui-text[site.locale].comments_labe
         {% endif %}
       </section>
     {% when "custom" %}
+      <h4 class="page__comments-title">{{ comments_label }}</h4>
```

Comment:
Variable is incorrect. Should be `{{ site.data.ui-text[site.locale].comments_label | default: "Leave a Comment" }}`